pkgs/top-level/all-packages.nix

```diff
@@ -193,6 +193,7 @@ with pkgs;
       pythonInterpreter = "${python3.withPackages (ps: [ ps.pyelftools ])}/bin/python";
       autoPatchelfScript = ../build-support/setup-hooks/auto-patchelf.py;
     };
+    meta.platforms = lib.platforms.linux ++ lib.platforms.freebsd;
```

Comment:
Is this necessary?  I already see FreeBSD in there:

```
nix-repl> autoPatchelfHook.meta.platforms
[
  "aarch64-linux"
  "armv5tel-linux"
  "armv6l-linux"
  "armv7a-linux"
  "armv7l-linux"
  "i686-linux"
  "loongarch64-linux"
  "m68k-linux"
  "microblaze-linux"
  "microblazeel-linux"
  "mips-linux"
  "mips64-linux"
  "mips64el-linux"
  "mipsel-linux"
  "powerpc64-linux"
  "powerpc64le-linux"
  "riscv32-linux"
  "riscv64-linux"
  "s390-linux"
  "s390x-linux"
  "x86_64-linux"
  "i686-freebsd"
  "x86_64-freebsd"
]
```

Reply:
Indeed. It is gone.